client/package.json

```diff
@@ -8,6 +8,7 @@
     "classnames": "^2.2.6",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
+    "lodash": "^4.17.15",
```

Comment:
`lodash-es` has been merged in previous PRs. `lodash` library is loads complete file even if we are using a single function. This issue has been solved in `lodash-es` package.

Reply:
updated to lodash-es